server/routes/bookedtoday/unexpectedArrivals/multipleExistingRecordsFoundController.test.ts

```diff
@@ -0,0 +1,112 @@
+import type { Express } from 'express'
+import request from 'supertest'
+import { SexKeys } from 'welcome'
+import * as cheerio from 'cheerio'
+import { appWithAllRoutes, flashProvider, signedCookiesProvider } from '../../__testutils/appSetup'
+
+import Role from '../../../authentication/role'
+import config from '../../../config'
+import { ExpectedArrivalsService } from '../../../services'
+
+jest.mock('../../../services/expectedArrivalsService')
+const expectedArrivalsService = new ExpectedArrivalsService(null, null) as jest.Mocked<ExpectedArrivalsService>
+
+const searchDetails = {
+  firstName: 'Jamie',
+  lastName: 'Smyth',
+  dateOfBirth: '1973-01-08',
+}
+
+const potentialMatches = [
+  {
+    firstName: 'James',
+    lastName: 'Smyth',
+    dateOfBirth: '1973-01-08',
+    prisonNumber: 'A1234BC',
+    pncNumber: '11/5678',
+    croNumber: '12/0000',
+    sex: SexKeys.MALE,
+  },
+  {
+    firstName: 'Jim',
+    lastName: 'Smith',
+    dateOfBirth: '1983-01-08',
+    sex: SexKeys.MALE,
+  },
+]
+let app: Express
+
+beforeEach(() => {
+  config.confirmNoIdentifiersEnabled = true
```

Comment:
don't need this? this is only required for the feature toggle around when no identifiers provided scenario

Reply:
fair point